### PR TITLE
Feature: Support upgradeable contracts

### PR DESCRIPTION
**Pull request by Anthares101**
## Description

- Update alastria-identity-lib dependency to point to [support-upgradeable-contracts](https://github.com/alastria/alastria-identity-lib/tree/feature/support-upgradeable-contracts) where the smart contracts addresses and  ABI have been updated (Must be reverted to master or develop once the we merge the changes).
- Update some keystores to work with the new smart contracts

### CODE REVIEW

**Comment by VictorNS69** [comment]
This PR fixes https://github.com/alastria/alastria-identity-lib/issues/116, does the alastria-identity-lib repo need a PR?

**Comment by Anthares101** [comment]
@VictorNS69 yes, the alastria-identity-lib repo will need a PR if we want to point the examples dependencies to the master branch again

File: package.json [hunk 1]
```diff
@@ -27,7 +27,7 @@
   },
   "homepage": "https://github.com/alastria/alastria-identity-example#readme",
   "dependencies": {
-    "alastria-identity-lib": "github:alastria/alastria-identity-lib#master",
+    "alastria-identity-lib": "github:alastria/alastria-identity-lib#feature/support-upgradeable-contracts",
```
Comment (VictorNS69) [inline package.json]:
```suggestion
    "alastria-identity-lib": "github:alastria/alastria-identity-lib#develop",
```
We should update the develop branch in _alastria-identity-lib_ in order to use the existing branch
- [x] Alastria-identity-lib updated

File: exampleFirstEntity/1.createEntityAlastriaID.js [hunk 2]
```diff
@@ -16,7 +16,7 @@ const keyDataEntity1 = fs.readFileSync(
 )
 const keystoreDataEntity1 = JSON.parse(keyDataEntity1)
 const keyDataAdmin = fs.readFileSync(
```
Comment (VictorNS69) [inline exampleFirstEntity/1.createEntityAlastriaID.js]: I think it is the time to change some variables in this file. "admin" no longer exists. Lets call it "firstIdentity". Please check the other files in `exampleFirstEntity`

**Review by VictorNS69 — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (Anthares101) [reply package.json]: Ok i was wrong sorry, what we really need is to update the `ContractInfo.md` file in the `alastria-identity` respository with the new smart contracts addresses. Once that is done, the library should be able to get the updated information

File: exampleFirstEntity/1.createEntityAlastriaID.js (hunk 2 above)
Comment (Anthares101) [reply exampleFirstEntity/1.createEntityAlastriaID.js]: I refactored the code, take a look if you can to make sure the changes to some README files are correct too. I also tested the examples after the changes and looks like everything is working as it should.

**Comment by VictorNS69** [comment]
This PR is blocked until we update the lib (https://github.com/alastria/alastria-identity-lib/issues/116)

File: package-lock.json [hunk 3]
```diff
@@ -5439,9 +5571,9 @@
       "integrity": "sha512-LKYU1iAXJXUgAXn9URjiu+MWhyUXHsvfp7mcuYm9dSUKK0/CjtrUwFAxD82/mCWbtLsGjFIad0wIsod4zrTAEQ=="
     },
     "y18n": {
-      "version": "4.0.0",
-      "resolved": "https://registry.npmjs.org/y18n/-/y18n-4.0.0.tgz",
-      "integrity": "sha512-r9S/ZyXu/Xu9q1tYlpsLIsa3EeLXXk0VwlxqTcFRfg9EhMW+17kbt9G0NrgCmhGb5vT2hyhJZLfDGx+7+5Uj/w=="
+      "version": "4.0.1",
```
Comment (VictorNS69) [inline package-lock.json:5271]: Closes #108

**Review by VictorNS69 — APPROVED** [review]

**Comment by VictorNS69** [comment]
Now this PR is ready, right? @anthares101  :rocket:

**Comment by Anthares101** [comment]
> Now this PR is ready, right? @anthares101

Yeah!!! :rocket:

**Pull request merged**